tests_localhost/test_convert.py

```diff
@@ -0,0 +1,95 @@
+# Copyright Contributors to the Packit project.
```

Comment:
would be nice to have only a single test suite

Reply:
> We can switch to this approach completelly if we want.

Can I remove the old one?

Or, have only one test case to check the containerisation?